client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -192,4 +200,50 @@ final case class Client(open: Service[Request, DisposableResponse], shutdown: Ta
     shutdown.run
 }
 
+<<<<<<< HEAD
+object Client {
+  /** Creates a mock client from the specified service.  Useful for generating
+    * pre-determined responses for requests in testing.
+    * 
+    * @param service the service to respond to requests to this client
+    */
+  def mock(service: HttpService): Client = {
```

Comment:
[Like this](http://stackoverflow.com/questions/26177797/how-to-use-sbt-for-interdependent-projects-in-different-configurations), which feels like a hack.

I like the idea of a testkit, but it could quickly become a ball of mud: server stuff, client stuff, maybe blaze and servlet stuff?  Then how useful is it without scalatest or scalacheck or specs2 integrations?